frontend/src/admin/manage/users/create-user.js

```diff
@@ -1,9 +1,6 @@
-import {inject} from 'aurelia-framework';
-import { DialogController } from 'aurelia-dialog';
```

Comment:
not sure if we use DialogCtrl in other places, but if we're not
please remove any reference of it from `main.js` and `aurelia.json`